mkt/developers/templates/developers/apps/status.html

```diff
@@ -372,14 +372,17 @@ <h2 id="low-memory-devices">{{ _("Low-Memory Devices") }}</h2>
             <p><a class="test-plan-download" href="{{ test_plan.preload_test_plan_url }}" download>
               {{ _('View your most recently submitted test plan') }}</a></p>
           {% endif %}
-          <p class="listing-footer">
-            {% set btn_txt = _('Learn how to get your app pre-installed on Firefox OS devices') %}
+         <p class="listing-footer">
+            {% set lnk_txt = _('Learn how') %}
+            {% set lbl_txt = _('to get your app pre-installed on Firefox OS devices') %}
             {% if test_plan %}
               {% set btn_txt = _('Update your test plan') %}
+              {% set lbl_txt = _('Update your test plan') %}
             {% endif %}
-            <a class="button" href="{{ addon.get_dev_url('preload_' + next_step_suffix) }}">
-              {{ btn_txt }}
+            <a href="{{ addon.get_dev_url('preload_' + next_step_suffix) }}">
+              {{ lnk_txt }}
```

Comment:
This is looking much better with the localisation in, but I think we can make it a little simpler, by just removing `{{ lbl_txt }}` in the case of a test plan, too me it doesn't add anything. If we did that we'd go from:

https://www.dropbox.com/s/q3e19awf7qaue1w/Screenshot%202015-02-27%2010.00.37.png?dl=0

To just:

https://www.dropbox.com/s/yoij3yyllmqp6nf/Screenshot%202015-02-27%2010.00.57.png?dl=0

Getting close, thanks for the work so far!